aidial_adapter_vertexai/app.py

```diff
@@ -18,16 +21,51 @@
 DEFAULT_REGION = get_env("DEFAULT_REGION")
 GCP_PROJECT_ID = get_env("GCP_PROJECT_ID")
 
+
+def _init_vertexai_client() -> None:
+    vertexai.init(project=GCP_PROJECT_ID, location=DEFAULT_REGION)
```

Comment:
Please preserve the original comment for `init_vertex_ai`

Reply:
Done